backend/users/models.py

```diff
@@ -1,15 +1,81 @@
 from django.contrib.auth.base_user import AbstractBaseUser
 from django.contrib.auth.models import PermissionsMixin
+from django.core import exceptions
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from model_utils import Choices
+from pycountry import countries
 
 from .managers import UserManager
 
+COUNTRIES = [{"code": country.alpha_2, "name": country.name} for country in countries]
+EU_COUNTRIES = (
+    "AT",
+    "BE",
+    "BG",
+    "CY",
+    "CZ",
+    "DK",
+    "EE",
+    "FI",
+    "FR",
+    "DE",
+    "GR",
+    "HU",
+    "HR",
+    "IE",
+    "IT",
+    "LV",
+    "LT",
+    "LU",
+    "MT",
+    "NL",
+    "PL",
+    "PT",
+    "RO",
+    "SK",
+    "SI",
+    "ES",
+    "SE",
+    "GB",
+)
+
+[
+    {"code": country.alpha_2, "name": country.name}
+    for country in countries
+    if country.alpha_2 in EU_COUNTRIES
+]
+
 
 class User(AbstractBaseUser, PermissionsMixin):
+
+    image = models.ImageField(_("image"), upload_to="user_image", null=True, blank=True)
     username = models.CharField(_("username"), max_length=100, null=True, blank=True)
     email = models.EmailField(_("email address"), unique=True)
 
+    first_name = models.CharField(_("first name"), max_length=30, blank=True)
+    last_name = models.CharField(_("last name"), max_length=150, blank=True)
+
+    GENDERS = Choices(("male", _("Male")), ("female", _("Female")))
+    gender = models.CharField(_("gender"), choices=GENDERS, max_length=10, blank=True)
+    date_birth = models.DateField(_("date of birth"), null=True)
+    open_to_recruiting = models.BooleanField(_("open to recruiting"), default=False)
+    open_to_newsletter = models.BooleanField(_("open to newsletter"), default=False)
+
+    business_name = models.CharField(_("business name"), max_length=150, blank=True)
+    fiscal_code = models.CharField(_("fiscal code"), max_length=16, blank=True)
+    vat_number = models.CharField(_("VAT number"), max_length=22, blank=True)
+    phone_number = models.CharField(_("phone number"), max_length=20, blank=True)
+
+    # electronic invoicing ita
+    recipient_code = models.CharField(_("recipient code"), max_length=7, blank=True)
+    pec_address = models.EmailField(_("PEC"), blank=True)
+
+    address = models.TextField(_("address"), blank=True)
+    country = models.CharField(
+        choices=COUNTRIES, max_length=50, verbose_name=_("country"), null=True
```

Comment:
Maybe `blank=True` instead of null? As it seems more fitting to leave it empty (adding default='')

Reply:
oks